ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,359 @@
+"""
+Merge DL1-files from stage1-process tool
+"""
+from pathlib import Path
+import sys
+import tables
+import os
+from argparse import ArgumentParser
+import numpy as np
+
+from tqdm import tqdm
+
+from ..core import Provenance
+from ctapipe.core import Tool, traits
+from traitlets import List
+from ctapipe.instrument import SubarrayDescription
+
+import warnings
+
+warnings.filterwarnings("ignore", category=tables.NaturalNameWarning)
+
+PROV = Provenance()
+
+required_nodes = [
+    "/dl1/monitoring/subarray/pointing",
+    "/dl1/monitoring/telescope/pointing",
+    "/dl1/service/image_statistics",
```

Comment:
I don't think these image statistics should be required. There are "nice to have"

Reply:
It was very unconvenient from me here to use the name of the list 'required_nodes', because it was supposed to have all nodes in it and then just do certain stuff if its e.g. a simu file or so... I will rename it to have a 'all_nodes' -list and then create a set of it ('required_nodes') with all the given requirements like --skip-something in it.